mino/minogrpc/tracing/tracers.go

```diff
@@ -0,0 +1,77 @@
+package tracing
+
+import (
+	"io"
+	"sync"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	_ "github.com/uber/jaeger-client-go"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+)
+
+var (
+	ProtocolTag = "protocol"
+)
+
+type key int
+
+const ProtocolKey key = iota
+
+type tracer struct {
+	addrTracer map[string]tracerCloser
+	sync.Mutex
+}
+
+type tracerCloser struct {
```

Comment:
naming: this is not a structure that can close or closes tracers, it's a "closable" tracer, right?
Why not just call it `closableTracer` or `tracerWrapper` ?